docker/cp-test

```diff
@@ -236,28 +160,17 @@ OPTIONS:
   -r [filter]       run spec test suite (implies -d); may be filtered by test
                     suite and name
   -H                run spec tests in "hosted" mode (implies -r and -d)
-  -u                run unit test suite N number of times, where N is number
-                    of times u was specifed in the arguments
-  -l                run the linters against the code
   -s                run a bash shell when done
-  -b <task>         execute the specified buildr task (deprecated)
   -c <ref>          git reference to checkout
   -j <version>      use a specific Java version instead of the auto-detected default
   -v                enable verbose/debug output
-  -i                internationalization - validate translation files
   -k                run cloud registration spec test suite (only for "hosted" mode)
-  -n                uploading PR/Branch on SonarQube server
-                    Below arguments are passed:
-                    1) SONAR_HOST_URL
-                    2) BRANCH_NAME, branch to be analyzed
-                    3) CHANGE_ID, pull request ID (when Uploading PR)
-                    4) CHANGE_TARGET, pull request target branch (when Uploading PR)
   -a <arguments>    extra arguments to pass to the Candlepin deploy script (implies -d)
 HELP
 }
 
 ARGV=("$@")
-while getopts ":dtRqrHulskib:c:a:vj:n" opt; do
```

Comment:
A couple issues with the colons here: a letter is supposed to be followed by a colon when that option requires an argument (OPTARG). So a) by removing `b`, you should also remove the colon after it, because now it is right after option `k`, which doesn't need it. Also, when removing `n`, you also removed the colon before it, which belonged to `j`, so that needs to be added back